### PR TITLE
Regenerate Spanner Gapic to use new HTTP headers.

### PR DESCRIPTION
**Pull request by landrito**
Updates #1268

cc @bjwatson @swcloud

### CODE REVIEW

File: google-cloud-spanner/lib/google/spanner/v1/spanner_pb.rb [hunk 1]
```diff
@@ -10,7 +10,6 @@
 require 'google/protobuf/timestamp_pb'
 require 'google/spanner/v1/keys_pb'
 require 'google/spanner/v1/mutation_pb'
-require 'google/spanner/v1/query_plan_pb'
```
Comment (blowmage) [inline google-cloud-spanner/lib/google/spanner/v1/spanner_pb.rb]: Why was this require removed?

Comment (landrito) [inline google-cloud-spanner/lib/google/spanner/v1/spanner_pb.rb]: This was directly from the proto compiler. I'll do some digging when I get back from lunch.

Comment (blowmage) [inline google-cloud-spanner/lib/google/spanner/v1/spanner_pb.rb]: this file is also required in `result_set_pb`, which is the next file required. This is probably okay.

**Comment by blowmage** [comment]
Ah, I forgot that spanner also creates an instance of OperationsClient, and like Speech this PR depends on google-gax 0.8.1.

**Pull request merged**